### DISCUSSION TITLE
HELP! How to git pull on a plane that blocks ssh connections?

### DISCUSSION THREAD

**Discussion by Tecktron** [community/community#34026] [Other Feature Feedback, Questions, & Ideas]
Travelling and on a plane's wifi, trying to `git pull` to grab the latest changes.
After a very long wait time, the connection was reset by peer, so obviously ssh is being blocked and isn't going to work.
Tried good-old HTTPS since it was still seems available on the UI, but that is a VERY misleading lie that it seems they forgot to update when they starting blocked these connections (update your UI github, come on!), so no luck there either.
Tried to do the SSH over HTTPS as suggested in the help docs, but same thing as ssh, doesn't work. 
What are my other alternatives here? Am I just sh!t out of luck trying to connect to github without ssh?
I'm incredibly frustrated by this, like WTF am I paying for if I cannot access my code?
Please help!

**Comment by triphora** [comment]
You can use the [GitHub CLI](https://cli.github.com/) to pull over HTTP.
First, run [`gh auth setup-git`](https://cli.github.com/manual/gh_auth_setup-git) to set up GitHub CLI as the credential helper.
Then, just pull as normal. `git pull https://github.com/Tecktron/my-repository.git [ref]`
